.github/workflows/dotnet-build-and-test.yml

```diff
@@ -8,11 +8,11 @@ name: dotnet-build-and-test
 on:
   workflow_dispatch:
   pull_request:
-    branches: ["main"]
+    branches: ["main", "feature-azure-functions"]
   merge_group:
-    branches: ["main"]
+    branches: ["main", "feature-azure-functions"]
   push:
-    branches: ["main"]
+    branches: ["main", "feature-azure-functions"]
```

Comment:
These changes need to be reverted prior to merging. They are just for testing the CI.

Reply:
Removed